### PR TITLE
KSTO streaming

### PR DESCRIPTION
**Pull request by drewvolz**
This PR adds streaming radio to the KSTO tab for both iOS and Android. Although it is summer and the stream is dead air (static), I promise it works.

@hawkrives your help would be greatly appreciated in putting the final styling touches on the [KSTO branch](https://github.com/StoDevX/AAO-React-Native/tree/ksto).

_(screenshots will be added later)_

### CODE REVIEW

File: .flowconfig [hunk 1]
```diff
@@ -30,7 +30,8 @@
 [libs]
 node_modules/react-native/Libraries/react-native/react-native-interface.js
 node_modules/react-native/flow
-flow-typed/
+flow-typed/aao
+flow-typed/npm
```
Comment (hawkrives) [inline .flowconfig:34]: I had to add these lines so that flow would find my types for the video player.

**Comment by hawkrives** [comment]
… do we not just want to merge this and add styling fixes before release?

**Comment by drewvolz** [comment]
I think styling fixes now would be good, but I’ll have you make that choice.

**Comment by hawkrives** [comment]
Well, I'm more likely to remember it once it's on master… so…

**Comment by hawkrives** [comment]
I'll merge it after I finish this current branch I have locally which might make merge conflicts.

**Review by hawkrives — APPROVED** [review]

**Review by elijahverdoorn — APPROVED** [review]

**Comment by hawkrives** [comment]
And look at that, no merge conflicts.

**Pull request merged**